ansible-scylla-node/tasks/common.yml

```diff
@@ -166,14 +166,42 @@
   ignore_errors: true
   #TODO: stop ignoring errors when the node_exporter_install script fix is available in all actual versions, resp. use only for < 5.0 / 2022
 
+- name: Re-populate package facts
+  package_facts:
+    manager: auto
+
 - name: node exporter setup from 5.0/2022
-  service:
-    name: scylla-node-exporter
-    state: started
-    enabled: yes
+  block:
+    - name: Start scylla-node-exporter service
+      service:
+        name: scylla-node-exporter
+        state: started
+        enabled: yes
+      register: _scylla_node_exporter_status
+
+    - name: Set custom scylla-node-exporter service configuration
+      block:
+        - name: Get {{ scylla_package_prefix }}-node-exporter environment configuration file name
+          set_fact:
+            # 'EnvironmentFiles' has the following format: '<file name> (ignore_errors=[yes|no])'
+            _scylla_node_exporter_env_config_file: "{{ _scylla_node_exporter_status.status.EnvironmentFiles.split(' ')[0] }}"
+
+        - name: Set a custom configuration in {{ _scylla_node_exporter_env_config_file | default('a scylla-node-exporter configuration file') }}
```

Comment:
It is not always set - this block is conditional.

Reply:
> It is not always set - this block is conditional.

I meant that if the task was not skipped, this will always be set.

> And when it's skipped the name printout is still printed as follows:

Got it